### PR TITLE
Add AuthProfile and AzProfile to MergeConfig

### PR DESCRIPTION
**Pull request by sozercan**
Adds `AuthProfile` and `AzProfile` to MergeConfig

### CODE REVIEW

File: pkg/plugin/plugin.go [hunk 1]
```diff
@@ -58,6 +58,12 @@ func (p *plugin) MergeConfig(ctx context.Context, cs, oldCs *api.OpenShiftManage
 	if cs.Properties.ServicePrincipalProfile == nil {
 		cs.Properties.ServicePrincipalProfile = oldCs.Properties.ServicePrincipalProfile
 	}
+	if cs.Properties.AzProfile == nil {
```
Comment (jim-minter) [inline pkg/plugin/plugin.go]: please can you update the unit test in plugin_test.go?

**Comment by jim-minter** [comment]
thanks @sozercan makes sense, please can you fix up the 2 comments then will merge?

**Comment by sozercan** [comment]
thanks @jim-minter, updated

**Pull request merged**